src/test/java/seedu/address/logic/parser/AddTaskCommandParserTest.java

```diff
@@ -55,157 +58,81 @@
+        assertParseSuccess(parser, PREAMBLE_WHITESPACE+TASK_A_NAME + TASK_A_DEADLINE
+                + TASK_A_PROGRESS, new AddTaskCommand(
+                expectedTask));
+
+        // multiple task names - last name accepted
+        assertParseSuccess(parser, TASK_B_NAME+TASK_A_NAME + TASK_A_DEADLINE
+                + TASK_A_PROGRESS, new AddTaskCommand(
                 expectedTask));
 
-        // multiple project names - last name accepted
-        assertParseSuccess(parser, PROJECT_NAME_DESC_AMY + PROJECT_NAME_DESC_BOB + DEADLINE_DESC_B
-                        + REPOURL_DESC_B + PROJECT_DESCRIPTION_DESC_BOB + PROJECT_TAG_DESC_FRIEND + TASK_DESC_DG,
-                new AddCommand(
-                        expectedProject));
 
         // multiple deadlines - last deadline accepted
-        assertParseSuccess(parser, PROJECT_NAME_DESC_BOB + DEADLINE_DESC_A + DEADLINE_DESC_B
-                        + REPOURL_DESC_B + PROJECT_DESCRIPTION_DESC_BOB + PROJECT_TAG_DESC_FRIEND + TASK_DESC_DG,
-                new AddCommand(
-                        expectedProject));
-
-        // multiple repoUrl - last repoUrl accepted
-        assertParseSuccess(parser, PROJECT_NAME_DESC_BOB + DEADLINE_DESC_B + REPOURL_DESC_A
-                        + REPOURL_DESC_B + PROJECT_DESCRIPTION_DESC_BOB + PROJECT_TAG_DESC_FRIEND + TASK_DESC_DG,
-                new AddCommand(
-                        expectedProject));
-
-        // multiple addresses - last address accepted
-        assertParseSuccess(parser, PROJECT_NAME_DESC_BOB + DEADLINE_DESC_B + REPOURL_DESC_B
-                + PROJECT_DESCRIPTION_DESC_AMY + PROJECT_DESCRIPTION_DESC_BOB + PROJECT_TAG_DESC_FRIEND
-                + TASK_DESC_DG, new AddCommand(expectedProject));
-
-        // multiple project tags - all accepted
-        Project expectedProjectMultipleTags = new ProjectBuilder(BOT).withTags(VALID_PROJECT_TAG_B,
-                VALID_PROJECT_TAG_A).withTasks(testTask1)
-                .build();
-        assertParseSuccess(parser, PROJECT_NAME_DESC_BOB + DEADLINE_DESC_B + REPOURL_DESC_B
-                + PROJECT_DESCRIPTION_DESC_BOB
-                + PROJECT_TAG_DESC_HUSBAND + PROJECT_TAG_DESC_FRIEND + TASK_DESC_DG, new AddCommand(
-                expectedProjectMultipleTags));
-
-        // multiple tasks - all accepted
-        Project expectedProjectMultipleTasks = new ProjectBuilder(BOT)
-                .withTasks(testTask1, testTask2)
-                .build();
-        assertParseSuccess(parser, PROJECT_NAME_DESC_BOB + DEADLINE_DESC_B + REPOURL_DESC_B
-                        + PROJECT_DESCRIPTION_DESC_BOB
-                        + PROJECT_TAG_DESC_FRIEND + PROJECT_TAG_DESC_HUSBAND + TASK_DESC_DG + TASK_DESC_MODEL,
-                new AddCommand(expectedProjectMultipleTasks));
-    }
+        assertParseSuccess(parser, TASK_A_NAME + TASK_B_DEADLINE+TASK_A_DEADLINE
+                + TASK_A_PROGRESS, new AddTaskCommand(
+                expectedTask));
```

Comment:
Why line is split into 2. It seems like it doesn't exceed the limit of a line of code.